api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
I'm not super familiar with queryToObject but can take a look. What's the specific issue? For exec, we expect multiple args of the same name, or at least we did when I originally wrote exec.

cc @smarterclayton